static/src/javascripts-legacy/projects/commercial/modules/dfp/empty-advert.js

```diff
@@ -1,13 +1,12 @@
 define([
-    'bonzo',
     'fastdom'
-], function (bonzo, fastdom) {
+], function (fastdom) {
     return emptyAdvert;
 
     function emptyAdvert(advert) {
         fastdom.write(function () {
             window.googletag.destroySlots([advert.slot]);
-            bonzo(advert.node).remove();
+            advert.node.parentNode.removeChild(advert.node);
```

Comment:
It should be! We are including set 3.6 which should contain all Element methods (see: https://polyfill.io/v2/docs/features/#default-sets)